### PR TITLE
Fixed bug with plugin icons not displaying after their instantiation

### LINKED ISSUE #383: Cannot instantiate diva's default plugins such as the DownloadPlugin
Diva's documentation specifies the following:
If a plugin is registered in the global context as part of the Diva namespace, it will be available after explicit instantiation. The following is an example of such instantiation

``` javascript
var diva = new Diva('diva-wrapper', {
                objectData: "https://images.simssa.ca/iiif/manuscripts/cdn-hsmu-m2149l4/manifest.json",
                plugins: [Diva.DownloadPlugin]
            });
```

This should result in displaying the download icon on top of every page, which is not the case.

### PR DESCRIPTION
**Pull request by zeyadhazem**
Problem was in instantiating the plugins. Changed the check for passed
plugins. Fixes #383

### CODE REVIEW

File: source/js/viewer-core.js [hunk 1]
```diff
@@ -1011,7 +1011,7 @@ export default class ViewerCore
 
     initPlugins ()
     {
-        if (!this.settings.hasOwnProperty.call('plugins'))
+        if (!this.settings.plugins)
```
Comment (ahankinson) [inline source/js/viewer-core.js]: I would go with `!this.settings.hasOwnProperty('plugins')`. The difference is that your returns `undefined` if it doesn't exist, but `hasOwnProperty` returns a boolean. 

The difference is largely academic, but if you get in the habit of checking for the existence of a property by looking for it with `hasOwnProperty` then you will avoid some bugs later on.

Comment (ahankinson) [inline source/js/viewer-core.js]: PS: What was written there originally was definitely wrong, though, so nice catch.

Comment (zeyadhazem) [inline source/js/viewer-core.js]: Yup makes sense! Thanks!

**Comment by ahankinson** [comment]
Thanks!

**Pull request merged**